drive/client.py

```diff
@@ -92,30 +92,30 @@ def _serialize_query_value(value):
 class Client:
     """Google Drive client"""
 
-    def __init__(self, credentials_path: Optional[str] = None, *, download_retries_count=5):
-        http = authorize_credentials(credentials_path)
-        self.service = discovery.build('drive', 'v3', http=http)
-        self.download_retries_count = download_retries_count
+    def __init__(self, credentials_path: Optional[str] = None, *, download_retries_count: int = 5) -> None:
+        http: httplib2.Http = authorize_credentials(credentials_path)
+        self.service: Any = discovery.build('drive', 'v3', http=http)
+        self.download_retries_count: int = download_retries_count
 
     @property
-    def _files(self):
+    def _files(self) -> Any:
         return self.service.files()
 
     @property
-    def _permissions(self):
+    def _permissions(self) -> Any:
         return self.service.permissions()
 
-    def create_folder(self, name: str, parent_id: Optional[str] = None):
+    def create_folder(self, name: str, parent_id: Optional[str] = None) -> File:
```

Comment:
Isn’t this annotation redundant with the `return cast(...)`?

Reply:
No, not in MyPy (I *think* Pyright can infer this). Try it out:

```python
from typing import assert_type

client = Client()
assert_type(client.create_folder('whatever', 'abc123'), File)
```

Without the explicit return type, this will error because it thinks the type is `Any`.